### ISSUE TITLE
Installing conda-build==1.19 while conda pinned deletes $/HOME/miniconda/bin/activate

### ISSUE THREAD

**Issue by gmnash** [conda/conda#2918]
Due to the fix for issue https://github.com/conda/conda/issues/2524 not yet being released (https://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh points to 4.0.5 not 4.1.x) I am pinning conda to 3.19.3.  This worked up until recently (I just noticed the problem today).  The code below demonstrates how installing conda-build can delete $HOME/miniconda/bin/activate making it impossible to activate any environment.

```
#!/bin/bash
set -e

rm -rf $HOME/miniconda
rm -rf Miniconda-latest-Linux-x86_64.sh

wget https://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh
chmod +x ./Miniconda-latest-Linux-x86_64.sh

./Miniconda-latest-Linux-x86_64.sh -b -p $HOME/miniconda
$HOME/miniconda/bin/conda install -y conda=3.19.3
echo "conda ==3.19.3" >> $HOME/miniconda/conda-meta/pinned

find $HOME/miniconda -name activate
conda install conda-build==1.19 -y
find $HOME/miniconda -name activate
```

The first find call outputs this:

```
$HOME/miniconda/pkgs/conda-env-2.4.5-py27_0/bin/activate
$HOME/miniconda/bin/activate
```

The second find call outputs this:

```
$HOME/miniconda/pkgs/conda-env-2.4.5-py27_0/bin/activate
```

**Comment by gmnash** [comment]
Hello.  Any update on this?

**Comment by kalefranz** [comment]
Conda uninstalling itself was fixed on conda 4.3.
